### PR TITLE
Fix/fix perceptron issues

### PR DESCRIPTION
**Pull request by grg121**
Perceptron classifier should be in binary folder, not NN. Also I had to remove the old undocumented python example.

### CODE REVIEW

File: examples/undocumented/python/classifier_perceptron.py [hunk 1]
```diff
@@ -1,42 +0,0 @@
-#!/usr/bin/env python
```
Comment (karlnapf) [inline examples/undocumented/python/classifier_perceptron.py]: let's name the file consistently with the others... just `perceptron.sg` (and same for the data)

**Comment by karlnapf** [comment]
Cool!
Before this can be merged, the data file needs update, as does the submodule revision in this PR

**Comment by grg121** [comment]
Perfect! I'm waiting [this PR](https://github.com/shogun-toolbox/shogun-data/pull/142) to be merger in Shogun-data before I could add the data folder to **this** PR.

I do in this way because is said in [developing.md](https://github.com/shogun-toolbox/docs/blob/master/DEVELOPING.md) but maybe I could have done all in this PR, I'm not familiar with git submodules.

**Comment by karlnapf** [comment]
I think you can already change the submodule before it is merged....(not sure though)
It IS merged now however ;)

**Comment by grg121** [comment]
Okay, I think now everything is alright! Next time I'll do all faster haha

**Comment by karlnapf** [comment]
Your data submodule is wrong in  here: https://travis-ci.org/shogun-toolbox/shogun/jobs/337352894#L455

**Comment by karlnapf** [comment]
Pls check the travis builds yourself to see these errors in the future.

**Comment by grg121** [comment]
Hi! sorry, I didn't see that it had failed! this time one of the travis test have failed with: "The job exceeded the maximum time limit for jobs, and has been terminated" having been working for 50 minutes while the rest of test pass in less than 20 minuts... 

I'll check again if there are more problems...

**Comment by karlnapf** [comment]
Seems good now, thanks!
You can see here that the integration test is executed as well (using your data) https://travis-ci.org/shogun-toolbox/shogun/jobs/337548666#L4824

**Pull request merged**